src/CompDevSDKMain.c

```diff
@@ -4,10 +4,12 @@
 #include "display.h"
 
 TaskHandle_t BlinkTaskHandle;
+
 void BlinkTask(void *param)
 {
     (void)param;
     static uint32_t blinksNum = 0;
+    displayInit();
```

Comment:
O tym myślę, że porozmawiamy - przeniosłem na chwilę tutaj bo okazało się, że nie można wywoływać taskYIELD, delay itd. z freertosa poza taskiem. Wydaje mi się, że dobrym rozwiązaniem będzie dodanie taska inicjalizacyjnego, który po inicjalizacji (np. wyświetlacza) uruchomi docelowe taski. Ale chciałbym się upewnić, czy to dobre podejście.

`void initTask()
{
   displayInit();
   initInnePierdoly();
   xStartTask(Blink)
   ...
}`